sensor/AdblockPlugin.js

```diff
@@ -89,27 +91,40 @@ class AdblockPlugin extends Sensor {
     }
 
     async getAdblockConfig() {
-      const json = await rclient.getAsync(configKey);
+      const result = {};
       try {
-        if (json == null) {
-          const result = {};
-          log.info(`Load config list from bone: ${configlistKey}`);
-          const data = await bone.hashsetAsync(configlistKey);
-          const arr = JSON.parse(data);
-          if (Array.isArray(arr)) {
-            for (var i=0; i<arr.length; i++) {
-              result[arr[i]] = "on";
+        log.info(`Load config list from bone: ${configlistKey}`);
+        const data = await bone.hashsetAsync(configlistKey);
+        //const data = "{\"ads\": {\"default\": \"true\"}, \"ads-adv\":{}}";
+        const adlist = JSON.parse(data);
+        if (!platform.getName() == 'gold') {
```

Comment:
use platform feature check code instead of if statement, so that it can support future platforms.

Example:
https://github.com/firewalla/firewalla/blob/master/platform/gold/GoldPlatform.js#L216